packages/tools/client-debug-view/src/components/AudienceView.tsx

```diff
@@ -166,16 +166,34 @@ function HistoryView(props: HistoryViewProps): React.ReactElement {
 		const changeTimeStamp = new Date(changeEntry.timestamp);
 		const wasChangeToday = nowTimeStamp.getDate() === changeTimeStamp.getDate();
 
+        const accordianBackgroundColor: IStackItemStyles = {
+            root: {
+                background: changeEntry.changeKind === "added" ? "lightgreen" : "orange",
+                padding: 3
+            }
+        }
+
 		historyViews.push(
-			<li key={`${changeEntry.clientId}-${changeEntry.changeKind}`}>
-				<b>Client ID: </b>
-				{changeEntry.clientId}
-				<br />
-				<b>Time: </b>{" "}
-				{wasChangeToday ? changeTimeStamp.toTimeString() : changeTimeStamp.toDateString()}
-				<br />
-				<b>Type: </b> {changeEntry.changeKind}
-			</li>,
+            <Stack horizontal={true} styles={accordianBackgroundColor}>
+                <StackItem>
+                    <IconButton
```

Comment:
I'm not sure this needs to be a button. Is there just an `Icon` component we can use?